### PR TITLE
Top coder logo not being displayed fix

### LINKED ISSUE #22: Top coder and URIOJ image not being displayed in some cards in coding calender home activity
**Actual Behaviour**
In the coding calender home activity, some cards showing the contests from "Top coder" and "URIOJ" do not have an image. It looks odd.

**Expected Behaviour**
Images should be displayed in the card.

**Steps to reproduce it**
Just click on "Coding calender" in the hamburger menu.

**Screenshots of the issue**
screenshot_20171210-034311

**Would you like to work on the issue?**
Yes i would like to work on the issue. I think the problem is in the code part that decides which contest is from which site. I can solve it.

### PR DESCRIPTION
**Pull request by Kartik2016**
Fix #22 

Changes: Topcoder image is now properly displayed in the coding calender.

### CODE REVIEW

**Comment by Kartik2016** [comment]
Done.

File: app/src/main/java/com/iitp/njack/iitp_connect/CodingCalendar/Adapters/CodingCalendarAdapter.java [hunk 1]
```diff
@@ -1,116 +1,119 @@
...
+
+        private void setImageViewUsingUrl(String url) {
+            URL urlPlatform;
+            try{
+                urlPlatform = new URL(url);
+                //get the platform of the contest.
+                String platformString = urlPlatform.getHost();
+                Log.v("ContestRecyclerViewAdap",platformString);
+
+                if (platformString.equals("www.topcoder.com")||platformString.contains("topcoder")){
```
Comment (srv-twry) [inline app/src/main/java/com/iitp/njack/iitp_connect/CodingCalendar/Adapters/CodingCalendarAdapter.java]: Here the if statement doesn't need the first check now, only the ```platformString.contains("topcoder")``` should be enough.

File: app/src/main/java/com/iitp/njack/iitp_connect/CodingCalendar/Adapters/CodingCalendarAdapter.java [hunk 2]
```diff
@@ -1,116 +1,119 @@
...
+                    contestPlatform.setImageResource(R.mipmap.topcoder_logo);
+                }else if (platformString.equals("www.codechef.com")){
```
Comment (srv-twry) [inline app/src/main/java/com/iitp/njack/iitp_connect/CodingCalendar/Adapters/CodingCalendarAdapter.java]: Also kindly change all the other ```.equals``` call to ```.contains``` call

**Comment by srv-twry** [comment]
@Kartik2016 Squash your commits here also.

**Comment by Kartik2016** [comment]
On it!

**Comment by Kartik2016** [comment]
Done!

**Pull request merged**